Dockerfile.template.erb

```diff
@@ -154,7 +154,8 @@ RUN apt-get update \
                   $buildDeps \
  && rm -rf /var/lib/apt/lists/* \
 <% end %>
- && rm -rf /tmp/* /var/tmp/* /usr/lib/ruby/gems/*/cache/*.gem
+ && rm -rf /tmp/* /var/tmp/* /usr/lib/ruby/gems/*/cache/*.gem \
+ && rm -rf /usr/lib/ruby/gems/2.*/gems/fluentd-*/test
```

Comment:
Could you merge this line into previous `rm -rf`?

Reply:
Done, thanks for reviewing!